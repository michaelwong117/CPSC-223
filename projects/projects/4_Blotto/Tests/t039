#!/bin/bash
# Larger example, score

trap "/usr/bin/killall -q -u $USER ./Blotto 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./Blotto ]; then
  echo './Blotto is missing or not executable'
  echo './Blotto is missing or not executable' 1>&2
  exit 1
fi


/c/cs474/bin/run -stderr=/dev/null ./Blotto /c/cs223/hw4/Tests/round_robin_25_5.in score 1 2 3 4 5 < /c/cs223/hw4/Tests/t039.in
