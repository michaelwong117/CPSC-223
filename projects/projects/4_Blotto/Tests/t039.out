  8.566 entry20
  8.421 entry35
  8.355 entry23
  8.316 entry13
  8.145 entry22
  8.118 entry37
  8.105 entry39
  8.092 entry14
  8.053 entry11
  8.039 entry17
  7.961 entry34
  7.921 entry15
  7.895 entry1
  7.882 entry28
  7.868 entry24
  7.803 entry12
  7.789 entry3
  7.789 entry32
  7.789 entry7
  7.724 entry6
  7.684 entry9
  7.658 entry4
  7.632 entry16
  7.632 entry33
  7.579 entry19
  7.421 entry25
  7.408 entry18
  7.395 entry29
  7.355 entry21
  7.289 entry38
  7.237 entry10
  7.237 entry8
  7.224 entry2
  7.197 entry26
  6.816 entry5
  6.618 entry30
  6.329 entry31
  5.921 entry36
  2.237 entry27
